app/views/users/sessions/_popup_login.html.erb

```diff
@@ -0,0 +1,59 @@
+<div id="loginModal" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
```

Comment:
.modal-content require a border: 0 to remove that slight extra white space at the bottom